### PR TITLE
Introducing husky a github pre-commit & pre-push hooks

### PR DESCRIPTION
**Pull request by frontr-uk**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/app/config.service.server.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import {Inject, Injectable, PLATFORM_ID} from '@angular/core';
 import {makeStateKey, TransferState} from '@angular/platform-browser';
 declare function require(name: string);
-const config = require('../../config');
+const configFile = require('../../config');
```
Comment (alectronic0) [inline src/app/config.service.server.ts]: not sure how i feel about the name change.

File: src/app/domain/components/questions/create/create.component.ts [hunk 2]
```diff
@@ -61,17 +58,17 @@ export class CreateQuestionsComponent implements OnInit {
     }
 
     submitCallback(values) {
-        values.subject && this.form.controls.subject.setValue(values.subject.trim());
-        values.question && this.form.controls.question.setValue(values.question.trim());
-        values.rounds = this.roundNumber;
+        // TODO FIX THIS ASSIGN
```
Comment (alectronic0) [inline src/app/domain/components/questions/create/create.component.ts]: should this be fixed before we merged (don't  think SSCS will be happy if they cannot create questions)

File: src/app/config.service.server.ts (hunk 1 above)
Comment (frontr-uk) [reply src/app/config.service.server.ts]: I was not happy about it also but the file had shadow names and for it was best idea... have a look at the file... it a puzzle :D

File: src/app/domain/components/questions/create/create.component.ts (hunk 2 above)
Comment (frontr-uk) [reply src/app/domain/components/questions/create/create.component.ts]: Agreed, I commented for temp, but I had to refactor it for lint to pass.

**Pull request merged**